src/application.ts

```diff
@@ -95,20 +95,22 @@ export class MyriadApiApplication extends BootMixin(
   }
 
   firebaseInit() {
-    if (
-      !config.FIREBASE_PROJECT_ID ||
-      !config.FIREBASE_CLIENT_EMAIL ||
-      !config.FIREBASE_PRIVATE_KEY
-    ) {
-      firebaseAdmin.initializeApp();
-    } else {
-      firebaseAdmin.initializeApp({
-        credential: firebaseAdmin.credential.cert({
-          projectId: config.FIREBASE_PROJECT_ID,
-          clientEmail: config.FIREBASE_CLIENT_EMAIL,
-          privateKey: config.FIREBASE_PRIVATE_KEY,
-        }),
-      });
+    if (!this.options.test) {
```

Comment:
to avoid nested if, how if write this line like this `if (this.options.test) return`